File: Assignment/chapter 3/excercise8.py

largest = None
smallest = None
sum1 = 0
product = 1


for number in range(3):
    userinput= int(input('Enter a number: '))
    if largest is None or userinput > largest:
        largest = userinput
        
    if smallest is None or userinput < smallest:
        smallest = userinput
        
    sum1 += userinput 
    product *= userinput
average = sum1 / 3
    
        
print('The largest number is', largest)
print('The smallest number is', smallest)

print ('The total addition of the number is', sum1)
print ('The product of the numbers is', product)
        

File: PycharmProjects/jaffa/chapter 3/Excercise8.java
import java.util.Scanner;
public class Excercise8{
public static void main(String...args){

	Scanner scanner = new Scanner(System.in);
	
	int sum = 0;
	int product = 1;
	int number1 = 0;
	int number2 = 0;
	int number3 = 0;
	int userInput = 0;
	
	
	for (int number = 1; number <= 3; number++){
		System.out.print("Enter number: ");
		 userInput = scanner.nextInt();
		
	if (userInput > largest)
		largest = userInput;
	if (userInput < largest)
		smallest = userInput; 		

			sum *= userInput;

	}
		
	System.out.println(largest);
	System.out.println(smallest);
	}

}